raiden/api/rest.py

```diff
@@ -561,7 +561,19 @@ def open(
             token_address=to_checksum_address(token_address),
             settle_timeout=settle_timeout,
         )
+        token = self.raiden_api.raiden.chain.token(token_address)
+        balance = token.balance_of(self.raiden_api.raiden.address)
         try:
+            if total_deposit is not None and total_deposit > balance:
+                error_msg =  'Not enough balance to deposit. {} Available={} Needed={}'.format(
```

Comment:
Here there is an extra space after `=`. I recommend installing `flake8` and running it to make sure that you got no style violations.  For that and more please read the [contributing](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md) guide.